File: Dataset/Leetcode/valid/6/104.py
 class Solution:
    def XXX(self, s, numRows):
        """
        :type s: str
        :type numRows: int
        :rtype: str
        """
        num = list(range(1,numRows+1))+list(range(numRows-1,1,-1))
        n = len(num)
        m = len(s)
        k = m//n
        r = m%n
      #生成一个序列标行标
        seq = k*num + num[0:r]       
        out = [""]*numRows        
        for i in range(m):
            out[seq[i]-1] += s[i]
            
        return ''.join(out)
        

File: Dataset/Leetcode/train/27/104.cpp
 class Solution {
public:
    int XXX(vector<int>& nums, int val) {
        vector<int> num;
        for(int i=0;i<nums.size();i++){
            if(nums[i]!=val)
                num.push_back(nums[i]);
        }
        int n;
        n=num.size();
        if(n!=0){
            int a[n];
            for(int i=0;i<n;i++)
            {
                a[i]=num[i];
            }
            nums=num;
            return n;
        }else{
            nums.clear();
            return 0;
        }
        

    }
};

